iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm

```diff
@@ -15,16 +15,16 @@
 #include "geometry/angles.hpp"
 
 namespace {
-CGFloat const kTurnsiPhoneWidth = 96;
```

Comment:
Sorry, how is this related to the declared goal of PR?

Reply:
It doesn't harm, and it shows to others how the new code should be written.